data/firehose/firehoseBlock.proto

```diff
@@ -25,8 +25,8 @@ message FirehoseBlock {
   map<string, RewardTx>   Rewards = 8;
   map<string, Receipt>    Receipts = 9;
   map<string, TxWithFee>  InvalidTxs = 10;
-  repeated Log            Logs = 11;
-  bytes                   SignersIndicesBitmap = 12;
+  map<string, Log>        Logs = 11;
+  repeated uint64         SignersIndicesBitmap = 12;
```

Comment:
why array of uint64 instead of byte array?

Reply:
There was a misunderstanding of this field meaning. Should remain repeated uint64, but renamed it to `SignersIndexes`